src/lib/pubkey/ecc_key/ecc_key.cpp

```diff
@@ -172,13 +172,7 @@ bool EC_PrivateKey::check_key(RandomNumberGenerator& rng, bool strong) const {
 }
 
 const BigInt& EC_PublicKey::get_int_field(std::string_view field) const {
-   if(field == "public_x") {
-      BOTAN_ASSERT_NOMSG(this->public_point().is_affine());
-      return this->public_point().get_x();
-   } else if(field == "public_y") {
-      BOTAN_ASSERT_NOMSG(this->public_point().is_affine());
-      return this->public_point().get_y();
-   } else if(field == "base_x") {
```

Comment:
Isn't that a minor semver violation? Perhaps at least throw a `Not_Implemented` with something akin to "not available anymore, sorry" for `public_y` and `public_x`?

Reply:
Maybe so but actually this was totally wrong - it would return the values in Montgomery form :/

I guess `Not_Implemented` is friendlier though